datasetUtils/createJob.go

```diff
@@ -29,7 +29,7 @@ Parameters:
 Returns:
 - jobId: A string representing the job ID if the job was successfully created, or an empty string otherwise
 */
-func CreateJob(client *http.Client, APIServer string, user map[string]string, datasetList []string, tapecopies *int) (jobId string) {
+func CreateArchivalJob(client *http.Client, APIServer string, user map[string]string, datasetList []string, tapecopies *int) (jobId string, err error) {
```

Comment:
why does tapecopy need to be a pointer?

Reply:
Originally, the code used the flags library that returned the flags' values as pointers. The person who worked on this tool before me just left the tapecopy as a pointer and wrote additional functions like that. One of the other PR's will fix this.